.git-authors

```diff
@@ -8,6 +8,7 @@ pairs:
   nw: Nathan Wang; nathan.wang2
   sc: Serena Cao; wcao0630
   mw: Matt Wills; mattwills8
+  md: Mark Philip Dawal; markphd 
```

Comment:
I think you and Matt don't have to put in the part after the `;`. Could you fix this for yourself and Matt? Make sure you can `git solo`.

Reply:
Should be good now. I tried `git solo` and it shows:
```
GIT_AUTHOR_NAME='Mark Philip Dawal'
GIT_AUTHOR_EMAIL='mark@forloop.me'
```
So I think everything is okay now? 😄